processNoise: 0.0001
sensorNoise: 0.0001

fx: 1.2
fy: -6.5